packages/turf-rhumb-bearing/index.js

```diff
@@ -52,7 +52,7 @@ export default function (start, end, options) {
  * const p2 = new LatLon(50.964, 1.853);
  * const d = p1.rhumbBearingTo(p2); // 116.7 m
  */
-function rhumbBearing(from, to) {
+function calculateRhumbBearing(from, to) {
```

Comment:
@stebogit I've been adding names to the export default function, in theory this shouldn't have any impact, however when bundling multiple modules sometimes the compile `dist/index.js` gets confused if another function has the same name as the main functions (`@turf/rhumbBearing`).

Have a look at the `dist/index.js` and you'll see how Rollup compiles the code.

Reply:
👍